Resources/Prototypes/Entities/Buildings/furniture.yml

```diff
@@ -80,3 +79,16 @@
     color: "white"
   - type: Icon
     state: wooden_chair
+
+- type: entity
+  name: Bed
```

Comment:
Remember that we dont use upper case for improper nouns in names anymore (this applies to the rest of the PR)

Reply:
I hate this